src/reformat.c

```diff
@@ -21,7 +21,7 @@ struct YUVBlock
     float v;
 };
 
-static avifBool avifGetRGBSpaceInfo(const avifRGBImage * rgb, avifRGBSpaceInfo * info)
+static avifBool avifGetRGBSpaceInfo(const avifRGBImage * rgb, avifRGBColorSpaceInfo * info)
```

Comment:
Nit: `avifGetRGBSpaceInfo` and `avifGetYUVSpaceInfo` probably shoud be renamed to match the new struct names.